introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-generation-flan-t5-Batch-Transform.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !pip install --upgrade sagemaker --quiet
</pre><p>not required.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>If using a combined artifact (want the apis to fetch model_uri as the prepacked artifact), it may be required. @hemant: please confirm.</p>

Reply:
<p>I confirmed it's not required!</p>